docs/articles/commands/custom_context_checks.md

```diff
@@ -10,8 +10,8 @@ A context check contains two important pieces:
 - The attribute that will be applied to the command. This contains parameters that will be passed to the executing check.
 - The check itself. This is the method that determines if the command can be executed.
 
-## Implementing a Context Check Attribute
-To create a context check, you will need to create a new attribute that inherits from `CheckBaseAttribute`. This attribute will be applied to the command method and will contain the parameters that will be passed to the check method.
+## Implementing a context check attribute
+To create a context check, you will need to create a new attribute that inherits from `ContextCheckAttribute`. This attribute will be applied to the command method and may contain metadata for the check method to use.
```

Comment:
This whole sentence doesn't really follow with the above; context checks are a two part system. Perhaps instead:
*Let's start by creating the attribute that will be applied to your command methods. You'll need to create a new class which inherits from `ContextCheckAttribute`. This class should also contain any properties that your check needs to determine whether or not to execute a command.*